developer-docs-site/docs/issues-and-workarounds.md

```diff
@@ -11,6 +11,16 @@ This page provides workarounds and answers for issues and questions that frequen
 If you found an issue that is not on this page, submit a [GitHub Issue](https://github.com/aptos-labs/aptos-core/issues). Make sure to follow the issue format used in this document. 
 :::
 
+## Compiler
+
+#### Description
+
+Your Move code will fail compile-time checks for transaction arguments and produce an error if the code contains an entry function with a return value or passes a structure as a parameter.
```

Comment:
Can you post the error here to make it clear?

Reply:
Hi @movekevin, please see the discussion I had on [Slack ](https://aptos-org.slack.com/archives/C036X27DZNG/p1679075937081269) with @wrwg about not documenting compiler error codes. Also, see [issue 6863](https://github.com/aptos-labs/aptos-core/issues/6863#issuecomment-1474626249).